addons/sms/models/sms_sms.py

```diff
@@ -72,6 +72,7 @@ def action_set_error(self, failure_type):
 
     def action_set_outgoing(self):
         self.state = 'outgoing'
+        self.failure_type = False
```

Comment:
Since we are now changing 2 different fields, could switch to the write syntax:

```py
self.write({
    'state': 'outgoing',
    'failure_type': False
})
```

I find it a little more explicit when writing several fields at once.

Reply:
Agree.
@awa-odoo I wrote these code becasue `action_set_error` was added recently, and I want to keep the similar coding style. But yes, `write` has better performance. I will change it.
Do you think I should also change the coding style of `action_set_error` in this task?

Reply:
Maybe just modify this one as the other is not really related to your changes here.
I'm not sure it has a very big impact on performances but it's a little nicer in any case.